pkg/bosh/manifest/dns.go

```diff
@@ -144,13 +145,15 @@ func (dns *boshDomainNameService) Reconcile(ctx context.Context, namespace strin
 		},
 	}
 
-	var mode int32 = 420
+	var corefileMode int32 = 420
```

Comment:
I know this was from the old value, but please say `0o644` instead for Unix permission bits.

Also curious why it's not `corefileMode := 0o644`, but I guess you need to (for some reason) specify the type explicitly? Same with `replicas`? (I know that it can't be `const` because we're taking the address…)

Reply:
Go allows both syntaxes:

```go
mode := int32(420)
var mode int32 = 420
```

Reply:
And yeah, 0644 makes much more sense...

Reply:
Fixed in https://github.com/cloudfoundry-incubator/cf-operator/pull/642/commits/e7b336078c1211fc56d12b195406f129cb0e2f27.